### DISCUSSION TITLE
未检测到主题和主题配置

### DISCUSSION THREAD

**Discussion by musnows** [Qexo/Qexo#393] [Q&A] [answered]
如题，我选择了自己的hexo配置文件仓库后，显示未检测到主题和主题配置（分支设置正确）

![image](https://github.com/Qexo/Qexo/assets/87506951/e792eebd-1633-44fe-8c8b-2567af0c5c04)

如下是该仓库的文件，请问是缺少了什么导致检测不到？

![image](https://github.com/Qexo/Qexo/assets/87506951/6e3f957a-7608-4fbb-80ca-8ff0298f09ba)

![image](https://github.com/Qexo/Qexo/assets/87506951/7bb15748-a0b1-41dd-955b-4267306fda72)

**Comment by am-abudu** [comment, marked as answer]
这是一个已知问题，在配置文件不规范的情况下会出现。

我不会对此进行修复，因为这并不影响使用，你可以直接点击”强制提交“

**Reply by musnows** [reply]
好的，hexo的主题配置的确有两个位置可选🤣
这个管理页面真的好棒，感谢作者！

**Reply by MemoryCollection** [reply]
你最后怎么解决的？

**Reply by am-abudu** [reply]
可以尝试强制提交或者使用Dev分支。这是已知问题

**Reply by musnows** [reply]
直接点击强制提交就可以了

**Comment by musnows** [comment]
> 你最后怎么解决的？
